### PR TITLE
fix the output to work in workflow

### PR DESCRIPTION
**Pull request by u9g**

### CODE REVIEW

File: start.js [hunk 1]
```diff
@@ -21,10 +21,6 @@ async function start () {
   const text = `minecraft_version=${wantedVersion}\nyarn_mappings=${version}`
   await fs.writeFile('gradle.properties', text)
   console.info('Config file written.')
-  const child = require('child_process').exec('gradle run')
```
Comment (rom1504) [inline start.js:24]: Can you explain why you don't run gradle run anymore ?

Comment (u9g) [reply start.js:24]: its better to put it in a seperate step that way we get output from it, running it like this i debugged for more hours im willing to admit to, without just being able to see the errors, while if i run gradle run seperately, you see the errors

Comment (rom1504) [reply start.js:24]: Then rename this script as "build-properties.js" or something, because it doesn't start anything anymore

Comment (u9g) [reply start.js:24]: done

**Comment by rom1504** [comment]
Ok can you adapt the readme ?

**Comment by u9g** [comment]
> Ok can you adapt the readme ?

done

**Review by rom1504 — APPROVED** [review]

**Pull request merged**